alembic/versions/70a659fcad0b_create_authors_and_books_tables.py

```diff
@@ -0,0 +1,46 @@
+"""Create authors and books tables
+
+Revision ID: 70a659fcad0b
+Revises: 
+Create Date: 2024-12-17 13:13:59.434432
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '70a659fcad0b'
+down_revision: Union[str, None] = None
```

Comment:
The `down_revision` is set to `None`. If this migration depends on a previous migration, you should specify the previous revision ID here. This ensures the migrations are applied in the correct order.